src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -91,6 +91,25 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
         }
     };
 
+    const closeCopyAndAlert = () => {
+        const url = new URL(window.location.href)
+        const urlParam = new URLSearchParams(url.search);
+        urlParam.delete('courseCode');
+        urlParam.delete('courseNumber');
+        urlParam.delete('deptLabel');
+        urlParam.delete('deptValue');
+        urlParam.delete('GE');
+        urlParam.delete('term');
+        urlParam.append('courseCode', String(section.sectionCode));
+        const new_url = `${url.origin.toString()}/?${urlParam.toString()}`;
+        navigator.clipboard.writeText(new_url.toString()).then(function() {
+            openSnackbar('success', 'Course Link Copied!');
+        }, function() {
+            openSnackbar('error', 'Fail to copy the link!');
+        });
```

Comment:
```suggestion
        navigator.clipboard.writeText(new_url.toString()).then(() => {
            openSnackbar('success', 'Course Link Copied!');
        }, () => {
            openSnackbar('error', 'Fail to copy the link!');
        });
```
I prefer the simpler arrow function syntax for cases like this.